src/components/Cards/index.js

```diff
@@ -0,0 +1,2 @@
+import Cards from "./component";
+export default Cards;
```

Comment:
요게 무슨 파일인가 했더니 Card 컴포넌트를 export 해주는 용도였군요?
이거는 아래와 같이 간단하게 쓸 수 있어요.
```suggestion
export { default as Card } from './Card';

```